setup.py

```diff
@@ -48,23 +48,25 @@ def extra(self):
     except ImportError:
         pass
     else:
+        # For cython test coverage:
+        # Uncomment the compiler_directives line below
+        # Run: python setup.py build_ext --force --inplace --define CYTHON_TRACE
         ext_modules = cythonize(
             [
+                'pydantic/class_validators.py',
+                'pydantic/dataclasses.py',
+                'pydantic/datetime_parse.py',
+                'pydantic/env_settings.py',
                 'pydantic/error_wrappers.py',
                 'pydantic/errors.py',
                 'pydantic/fields.py',
+                'pydantic/json.py',
                 'pydantic/parse.py',
-                'pydantic/validators.py',
-                'pydantic/types.py',
-                'pydantic/utils.py',
                 'pydantic/parse_files.py',
-                'pydantic/json.py',
-                'pydantic/datetime_parse.py',
-                'pydantic/dataclasses.py',
-                'pydantic/class_validators.py',
                 'pydantic/schema.py',
-                'pydantic/env_settings.py',
-                'pydantic/color.py',
```

Comment:
`color.py` makes heavy use of a namedtuple called RGBA -- https://github.com/cython/cython/issues/2649 explains why this is a problem (ultimately has the same root cause as issue 2552 referenced for `dataclass` above -- issues with cython and __annotations__). I couldn't find a nice workaround like I did for the `dataclass` issue above, so I just dropped color from being cythonized.